src/components/SnapshotDetails/tabs/SnapshotComponentsListRow.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import ExternalLink from '../../../shared/components/links/ExternalLink';
+import { RowFunctionArgs, TableData } from '../../../shared/components/table';
+import { useWorkspaceInfo } from '../../../utils/workspace-context-utils';
+import { commitsTableColumnClasses } from './SnapshotComponentsListHeader';
+
+export type SnapshotComponentTableData = {
+  metadata: { uid: string; name: string };
+  name: string;
+  containerImage: string;
+  application: string;
+  source?: { git?: { url: string; revision: string } };
+};
+
+const SnapshotComponentsListRow: React.FC<RowFunctionArgs<SnapshotComponentTableData>> = ({
+  obj,
+}) => {
+  const { workspace } = useWorkspaceInfo();
+  return (
+    <>
+      <TableData data-test="snapshot-component-list-row" className={commitsTableColumnClasses.name}>
+        <Link
+          to={`/application-pipeline/workspaces/${workspace}/applications/${obj.application}/component/${obj.name}`}
```

Comment:
Right now this link gives 404, but that should be handled with #763 right?

Reply:
I noticed that PR is using `/components/:componentName` format, so we might need to update to that